tests/helpers/census.go

```diff
@@ -0,0 +1,133 @@
+package helpers
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	censustest "github.com/vocdoni/davinci-node/census/test"
+	"github.com/vocdoni/davinci-node/circuits"
+	"github.com/vocdoni/davinci-node/crypto/csp"
+	"github.com/vocdoni/davinci-node/crypto/signatures/ethereum"
+	"github.com/vocdoni/davinci-node/state"
+	"github.com/vocdoni/davinci-node/types"
+	"github.com/vocdoni/davinci-node/web3"
+)
+
+func IsCSPCensus() bool {
+	cspCensusEnvVar := os.Getenv(CSPCensusEnvVarName)
+	return strings.ToLower(cspCensusEnvVar) == "true" || cspCensusEnvVar == "1"
+}
+
+func TestCensusOrigin() types.CensusOrigin {
+	if IsCSPCensus() {
+		return types.CensusOriginCSPEdDSABN254V1
+	} else {
+		return types.CensusOriginMerkleTreeOffchainDynamicV1
+	}
+}
+
+func TestWrongCensusOrigin() types.CensusOrigin {
+	if IsCSPCensus() {
+		return types.CensusOriginMerkleTreeOffchainStaticV1
+	} else {
+		return types.CensusOriginCSPEdDSABN254V1
+	}
+}
+
+func TestCensusWithRandomVoters(ctx context.Context, origin types.CensusOrigin, nVoters int) ([]byte, string, []*ethereum.Signer, error) {
```

Comment:
the `Test` prefix is confusing here (and other helpers here), should be just CensusWithRandomVoters (or, if the word Test is fundamental, then CensusWithRandomVotersForTest)